pyproject.toml

```diff
@@ -81,7 +81,7 @@ script = "devtasks:clean"
 help = "remove build/python artifacts"
 
 [tool.poe.tasks.coverage]
-cmd = "pytest --cov-report html --cov copier copier tests"
+cmd = "coverage run -m pytest -p pytest_cov --cov-report html --cov-report xml --cov-append"
```

Comment:
Ran this, but I don't get any xml or html files produced. Am I missing something? Maybe some docs will help.

Reply:
Sorry, bad version submitted, this should be fixed: it was missing the `--cov` but I also haven't kept the `coverage run -m` as it was not bringing so much benefit (on some pytest plugin I wrote, I wasn't able to have proper coverage with `pytest-cov` while it was working jsut fine with `coverage run` this is why I tried this prefix)